sdk/cosmosdb/cosmos/package.json

```diff
@@ -41,45 +46,67 @@
     "test": "npm run build:test && npm run unit-test && npm run integration-test",
     "unit-test:browser": "echo skipped",
     "unit-test:node": "echo skipped",
-    "unit-test": "npm run unit-test:node && npm run unit-test:browser",
-    "webpack:dev": "webpack -d",
-    "webpack": "webpack -p"
+    "unit-test": "npm run unit-test:node && npm run unit-test:browser"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/Azure/azure-cosmos-js"
+  },
+  "license": "MIT",
+  "tsdoc": {
+    "tsdocFlavor": "AEDoc"
+  },
+  "dependencies": {
+    "@azure/cosmos-sign": "1.0.2",
```

Comment:
Should always use `^caret` versions for our dependencies and devDependencies. Also make sure that any which are already used in other projects within the SDK are specified with the same version.

Reply:
Fixed.